deploy/downstream-prep.sh

```diff
@@ -66,3 +77,18 @@ sed -i "/MIG_UI_TAG/,/^ *[^:]*:/s/value: release-1.0/value: ${V1_0_IMG_MAP[ui_sh
 sed -i "/VELERO_PLUGIN_TAG/,/^ *[^:]*:/s/value: release-1.0/value: ${V1_0_IMG_MAP[plugin_sha]}/"              deploy/olm-catalog/mig-operator/v1.0.1/mig-operator.v1.0.1.clusterserviceversion.yaml
 sed -i "/VELERO_TAG/,/^ *[^:]*:/s/value: fusor-1.1/value: ${V1_0_IMG_MAP[velero_sha]}/"                       deploy/olm-catalog/mig-operator/v1.0.1/mig-operator.v1.0.1.clusterserviceversion.yaml
 sed -i "/VELERO_RESTIC_RESTORE_HELPER_TAG/,/^ *[^:]*:/s/value: fusor-1.1/value: ${V1_0_IMG_MAP[helper_sha]}/" deploy/olm-catalog/mig-operator/v1.0.1/mig-operator.v1.0.1.clusterserviceversion.yaml
+
+# Make 1.0.1 Downstream non-OLM changes
+sed -i s,quay.io,registry.redhat.io,g                                                                         deploy/non-olm/v1.0.1/operator.yml
+sed -i s,ocpmigrate,rhcam-1-0,g                                                                               deploy/non-olm/v1.0.1/operator.yml
+sed -i s,mig-operator:release-1\.0,openshift-migration-rhel7-operator:v1.0,g                                  deploy/non-olm/v1.0.1/operator.yml
```

Comment:
We can't pin the operator to a SHA. We're building it and we don't know what the SHA will be until it's done. I don't think there's a way for us to pin it to a SHA and release the operator.yml in a container.